server/proxy/pf_context.c

```diff
@@ -79,3 +79,32 @@ rdpContext* p_client_context_create(rdpSettings* clientSettings,
 	settings->RedirectClipboard = FALSE;
 	return context;
 }
+
+proxyData* pf_context_proxy_data_new()
```

Comment:
It is related to the PR because this struct contains the struct we're passing to the filters. Therefore, I had to change free(pdata) to a function that frees all of its members, including the connectionInfo strcut.